package.json

```diff
@@ -55,6 +55,7 @@
     "onWebviewPanel:java.welcome",
     "onCommand:java.classpathConfiguration",
     "onWebviewPanel:java.classpathConfiguration",
+    "onWebviewPanel:java.markdownPreview",
```

Comment:
what's this line for? @jdneo please take a look

Reply:
i don't think i edited this line but somehow it's showing up here, i can remove it if helps

Reply:
This line has been removed in a recent commit. Feel safe to remove it

Reply:
just removed